src/Microsoft.DotNet.Interactive.Http/HttpKernel.cs

```diff
@@ -202,6 +227,16 @@ async Task SendRequestAndHandleResponseAsync()
         {
             var response = await GetResponseWithTimingAsync(requestMessage, cancellationToken);
 
+            if(requestNode != null)
```

Comment:
The newer syntax `requestNode is not null` is generally preferable because, by avoiding the possibility of an operator overload causing a surprising behavior, it's a bit more robust.